kubemarine/procedures/config.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import json
+import sys
+from typing import List
+
+import yaml
+
+from kubemarine import plugins
+from kubemarine.core import static, utils
+
+HELP_DESCRIPTION = """\
+Print supported configurations of 3rd-parties with versions compatibility.
+
+How to use:
+
+"""
+
+
+def make_config() -> dict:
+    kubernetes_versions: dict = {}
+    for version, compatibility_map in static.KUBERNETES_VERSIONS['compatibility_map'].items():
+        kubernetes_version = kubernetes_versions.setdefault(version, {})
+
+        minor_version = utils.minor_version(version)
```

Comment:
I suggest to not print this column. Such it is almost always "supported" and looks confusing , since the point is to print supported only.

Reply:
Do you mean to do not print the version at all, if `supported: false`? https://github.com/Netcracker/KubeMarine/blob/main/kubemarine/resources/configurations/compatibility/kubernetes_versions.yaml#L2

Reply:
Done in https://github.com/Netcracker/KubeMarine/pull/613/commits/3590a955b60f4b10075e07fffa1256dc2fa810cb